src/plugins/vis_type_timeseries/public/application/components/custom_color_swatches.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export const COLOR_SWATCHES = [
+  '#0F1419',
```

Comment:
@cchaos  I want to know your opinion, do we now need this custom palette for TSVB?
Default looks not bad from my point of view. Probably now is a good time to remove custom COLOR_SWATCHES

![image](https://user-images.githubusercontent.com/20072247/84639576-3bb88d00-af01-11ea-8b2f-ddf934fee0c1.png)

Reply:
This is a good point, I kept them in order to keep kind of the same UX to the users that use TSVB a lot.

Reply:
Hi @alexwizp and @stratoula,

We should keep the custom `COLOR_SWATCHES` but can we use the `euiPaletteColorBlind()` instead? https://elastic.github.io/eui/#/utilities/color-palettes

We want to influence users to pick the right colors, so this palette provides them a good start. And it also works well with the dark theme.